packages/graphback-core/src/runtime/interfaces.ts

```diff
@@ -12,7 +12,10 @@ export interface GraphbackServiceConfigMap {
  * Contains resolver options
  */
 export interface GraphbackResolverOptions {
-  selectedFields: string[]
+  selectedFields: string[],
+  aggregations?: {
```

Comment:
I will add a comment to this field as it is not obvious what it does.

+100 to have it wrapped.

Reply:
The idea is that this will be a container for aggregations operations.

Reply:
Something along those lines will be okay?

Reply:
Perfect!